src/gtest/test_merkletree.cpp

```diff
@@ -1,9 +1,11 @@
 #include <gtest/gtest.h>
 
 #include "test/data/merkle_roots.json.h"
+#include "test/data/merkle_roots_new.json.h"
```

Comment:
I'm generally not a fan of calling things "new", since inevitably they become the old thing while still being called "new": https://en.wikipedia.org/wiki/New_College,_Oxford